### PR TITLE
(WIP) CNV-3890 Linking Import VMware VM module to Create VMs assembly

### PR DESCRIPTION
**Pull request by bgaydosrh**
Linking existing module to this assembly.

See Jira task: https://issues.redhat.com/browse/CNV-3890

Test Build: http://file.bos.redhat.com/bgaydos/021120/cnv/cnv_virtual_machines/cnv-create-vms.html

Tagging @alexxa for QE as she spotted this problem, as @ousleyp or @aburdenthehand for peer review and merge once Irina signs off.

Label for enterprise-4.3 and 4.4.

Thanks
Bob

### CODE REVIEW

**Comment by alexxa** [comment]
@bgaydosrh I don't know the feature details to validate it. @nellyc  who can proof read this doc chapter once more, not only this PR change, please?

**Comment by bgaydosrh** [comment]
Hi. Following up with @nellyc on this one. Any ideas who can help with QE?

Thanks
Bob

**Comment by nellyc** [comment]
Sorry, I missed the notification on this one. We need someone from IMS QE to review this one, I'll let them know

**Comment by bgaydosrh** [comment]
@nellyc - any updates on this?

**Comment by apinnick** [comment]
@bgaydosrh The SSH workaround does not apply to 4.4.

File: modules/cnv-importing-vmware-vm.adoc [hunk 1]
```diff
@@ -70,7 +70,7 @@ If you do not select a storage class, {CNVProductName} uses the default storage
 
 . In the *Advanced* screen, enter the *Hostname* and *Authorized SSH Keys* if you are using `cloud-init`.
 +
-You must xref:../../../cnv/cnv_virtual_machines/cnv-edit-vms.adoc#cnv-editing-vm-yaml-web_cnv-edit-vms[edit the virtual machine's YAML configuration] after importing it:
+You must edit the virtual machine's YAML configuration after importing it:
```
Comment (apinnick) [inline modules/cnv-importing-vmware-vm.adoc:73]: This part of the procedure (editing the VM yaml) should be removed. It was a workaround. The bug will be fixed when 4.4 is released.

**Comment by apinnick** [comment]
Question -- Why does "Importing VMware VMs" need to be added to "Creating VMs"? It already exists in "Importing VMs".

**Comment by bgaydosrh** [comment]
Thanks @apinnick. I am marking this WIP for now and will restart work in the 2.4 timeframe to so as you request. Appreciate the feedback.

Bob

**Comment by bgaydosrh** [comment]
Closing PR --- we will just be adding an XREF.

**Pull request closed without merging**